core/gui/src/app/workspace/service/joint-ui/joint-ui.service.ts

```diff
@@ -338,26 +278,6 @@ export class JointUIService {
       [`.${operatorAbbreviatedCountClass}`]: { text: abbreviatedText },
     });
   }
-
-  public changeOperatorEditingStatus(jointPaper: joint.dia.Paper, operatorID: string, users?: User[]): void {
-    if (users) {
-      const statusText = users[0].name + " is editing properties...";
-      const color = users[0].color;
-      jointPaper.getModelById(operatorID).attr({
-        [`.${operatorCoeditorEditingClass}`]: {
-          text: statusText,
-          fillColor: color,
-        },
-      });
-    } else {
-      jointPaper.getModelById(operatorID).attr({
-        [`.${operatorCoeditorEditingClass}`]: {
-          text: "",
-        },
-      });
-    }
-  }
```

Comment:
why is this not used? @Xiao-zhen-Liu is this for shared-editing to show other users' editing status?

Reply:
This was likely never used and introduced in #1676 by accident. The code for showing other user's editing status is in another file. We can remove it.